src/api/endpoints.ts

```diff
@@ -0,0 +1,27 @@
+const RESPONSE_LIMIT = 6;
+
+const URL = 'https://6450f8cbe1f6f1bb22a40fb5.mockapi.io/api/v1/';
```

Comment:
you can add transformRoute util like this

```
export const transformRoute = (route: string, value?: string): string =>
  route.replace(PARAM_PATTERN, value || EMPTY_STRING);
```

and use it in this way 

apiClient.get(
        transformRoute(ApiRoutes.GET_NOTE_BY_ID, requestId),
      );

where   GET_NOTE_BY_ID = '/notes/:id',

so you will have no need in this file and it will be more readable

just add an enum ApiRoutes, where you will describe all needed routes